pom.xml

```diff
@@ -45,8 +45,20 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>javax.validation</groupId>
+            <artifactId>validation-api</artifactId>
+            <version>2.0.1.Final</version>
```

Comment:
it seems like the version of the library can be provided with the BOM file, so no need to define it here

Reply:
done for spring-boot-starter-validation